03-Node-npm-modules/ver.0.2/downloableMovie v0.2.js

```diff
@@ -0,0 +1,107 @@
+var MovieObserver = function ( ) {};
```

Comment:
my bad. This folder is not supposed to be in this proyect. Will delete in next commit